src/Button/Button.js

```diff
@@ -1,56 +1,55 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { buttonVariants, buttonSizes, buttonTypes } from "./Button.levers";
+import React from 'react';
+import PropTypes from 'prop-types';
+import { buttonVariants, buttonSizes, buttonTypes } from './Button.levers';
 
 /**
  * Button Component
  *
  * HTML button element
  */
-export const Button = ({ className, children, onClick, ref, ...props }) => {
+export const Button = ({ className, children, onClick, ref, variant, size, type, isDisabled, ...props }) => {
```

Comment:
If you destructure each of the levers in the constructor you can keep the `{...props}` on the button HTML element which might come in handy. If you don't destructure the props arg then you can't use `{...props}` on the HTML element as React doesn't know what to do with props like `isDisabled: bool`